ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.17
+    tag: 1.8.3
```

Comment:
1.18.3 - Also should we update component.yml?

Reply:
I have addressed the comment :) Thank you :)